framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -1480,10 +1480,15 @@ public Void visitUnary(UnaryTree node, Void p) {
      * {@link #checkAssignability(AnnotatedTypeMirror, Tree)}.
      */
     @Override
-    public Void visitCompoundAssignment(CompoundAssignmentTree node, Void p) {
-        commonAssignmentCheck(node.getVariable(), node.getExpression(),
+    public Void visitCompoundAssignment(CompoundAssignmentTree compoundAssignmentTree, Void p) {
```

Comment:
All other methods use `node` for the first parameter. I'm not sure using the long name is helpful. (I see this addresses Javier's comment in an indirect way.)